### ISSUE TITLE
Error 500 creating posts on v2 (beta8)

### ISSUE THREAD

**Issue by enricodeleo** [WP-API/WP-API#1844]
Hi!
Just noticed that when I try to create a new post (both built-in or CPT), even if the content is created the api replies with error 500 (well, `500 hphp_invoke` since I am on HHVM).

This is the content of the post:

```json
{
    "title": "test",
    "content": "test content",
    "excerpt": "test example",
    "status": "publish"
}
```

The dashboard doesn't show anything abnormal. The content seems identical to one created via backend. 

Any hint?

**Comment by danielbachhuber** [comment]
> since I am on HHVM

Can you reproduce on a non HHVM install? And, can you show the full request and response?

**Comment by rmccue** [comment]
> 500 hphp_invoke

This usually indicates that HHVM couldn't actually run the script, so likely a parsing error or the file missing. What's in your log file? (`/var/log/hhvm/error.log` usually)

**Comment by enricodeleo** [comment]
@danielbachhuber @rmccue thanks for your hints. I tried on a regular PHP 5.6 install and that helped debugging the issue (HHVM doesn't help a lot imho).

Apparently the issue was related to another plugin (rest api filter), and removing it made the post working again both on PHP and HHVM. Strange fact is that other POSTs worked and the issue appeared only during post creation. Anyway, since filtering fields wanted in json is cleaner but not mandatory I just removed that plugin.

Thank you for your time and help! :)
